packages/features/bookings/Booker/components/EventMeta.tsx

```diff
@@ -68,7 +70,11 @@ export const EventMeta = () => {
           <EventTitle className="my-2">{event?.title}</EventTitle>
           {event.description && (
             <EventMetaBlock contentClassName="mb-8 break-words max-w-full max-h-[180px] scroll-bar pr-4">
-              <div dangerouslySetInnerHTML={{ __html: markdownToSafeHTML(event.description) }} />
+              {defaultEvents.find((defaultEvent) => defaultEvent.description === event.description) ? (
```

Comment:
Please check , if you can use "isDynamic" flag present in the event to determine which path to take than relying on defaultEvent description.